src/clue.js

```diff
@@ -1,94 +1,141 @@
+last_name:    "Plum",
+color:        "purple",
+description:  "Billionare video game designer",
+age:          22,
+image:        "https://metrouk2.files.wordpress.com/2016/07/professor-plum.jpg",
+occupation:   "Designer",
+}
+
+var missScarlet = {
+first_name:   "Kasandra",
+last_name:    "Scarlet",
+color:        "red",
+description:  "She is an A-list movie star with a dark past",
+age:          31,
+image:        "https://metrouk2.files.wordpress.com/2016/07/miss-scarlett.jpg",
+occupation:   "Actor",
+}
+
+var mrsPeacock = {
+first_name:   "Eleanor",
+last_name:    "Peacock",
+color:        "blue",
+description:  "She is from a wealthy family and uses her status and money to earn popularity",
+age:          36,
+image:        "https://metrouk2.files.wordpress.com/2016/07/mrs-peacock.jpg",
+occupation:   "Socialité",
+}
+
+var mrMustard = {
+first_name:   "Jack",
+last_name:    "Mustard",
+color:        "yellow",
+description:  "He is a former football player who tries to get by on his former glory",
+age:          62,
+image:        "https://metrouk2.files.wordpress.com/2016/07/colonel-mustard.jpg",
+occupation:   "Retired Football player",
+}
 
 // Weapons
 
-name: rope        ---   weight: 10
-name: knife       ---   weight: 8
-name: candlestick ---   weight: 2
-name: dumbbell    ---   weight: 30
-name: poison      ---   weight: 2
-name: axe         ---   weight: 15
-name: bat         ---   weight: 13
-name: trophy      ---   weight: 25
-name: pistol      ---   weight: 20
+var rope        = {weight: 10,}
+var knife       = {weight: 8,}
+var candlestick = {weight: 2,}
+var dumbbell    = {weight: 30,}
+var poison      = {weight: 2,}
+var axe         = {weight: 15,}
+var bat         = {weight: 13,}
+var trophy      = {weight: 25,}
+var pistol      = {weight: 20,}
 
 // Rooms
-name: Dinning Room
-name: Conservatory
-name: Kitchen
-name: Study
-name: Library
-name: Billiard Room
-name: Lounge
-name: Ballroom
-name: Hall
-name: Spa
-name: Living Room
-name: Observatory
-name: Theater
-name: Guest House
-name: Patio
-*/
+
+var rooms = [
+"Dinning Room",
+"Conservatory",
+"Kitchen",
+"Study",
+"Library",
+"Billiard Room",
+"Lounge",
+"Ballroom",
+"Hall",
+"Spa",
+"Living Room",
+"Observatory",
+"Theater",
+"Guest House",
+"Patio",
+];
 
 // Characters Collection
 var charactersArray = [];
 
+charactersArray.push(mrGreen, drOrchid, profPlum, missScarlet, mrsPeacock, mrMustard);
+console.log(charactersArray);
+
+
 // Rooms' Collection
 var roomsArray = [];
 
+roomsArray.push(rooms);
+console.log(roomsArray);
+
+
 // Weapons Collection
 var weaponsArray = [];
+
+weaponsArray.push(rope, knife, candlestick, dumbbell, poison, axe, bat, trophy, pistol);
+console.log(weaponsArray);
+
+
+// ITERATION 2
+
+/* Create a method randomSelector to randomly select one element from a card stack.
+The method should receive an array as an argument,
+and return randomly one of the elements of the array */
+
+random = function randomNum (array) {
```

Comment:
this is a really good use of a function but maybe you could have passed it as an anonymous. But really good structure here !